core/templates/services/contextual/logger.service.ts

```diff
@@ -24,24 +24,44 @@ import { Injectable } from '@angular/core';
 export class LoggerService {
   constructor() {}
 
+  /**
+   * Logs a message to console at the debug level.
+   * @param {string} msg - A message to log.
```

Comment:
Thanks! Please don't add types to the JSDocs, however, since they can easily become out-of-sync with the actual types the functions expect. Just a short description of what they're used for is good enough.

```suggestion
   * @param msg - A message to log.
```

Reply:
Hi @brianrodri. I had noticed the types were specified in the JSDocs on the examples given by @Showtim3 on issue #8668, so I added them, but I just pushed a commit to fix that. Thanks for pointing it out!